about.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&family=Lato:wght@400;700;900&family=Poppins:ital,wght@0,200;0,400;0,600;1,500&display=swap" rel="stylesheet">
+<link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="about.css">
+  <title>Concert Page</title>
+</head>
+<body>
+  <section class="introduction">
+    <div id="overlay">
+      <div id="close-container" onclick="menuClose()">
+        <a id="closer" onclick="menuClose()">
+          <img src="./images/Icon.png" alt="Close" onclick="menuClose()">
+        </a>
+      </div>
+      <div id="menucontainer">
+          <a href="" onclick="menuClose()">Home</a>
+          <a href="" onclick="menuClose()">About</a>
+          <a href="" onclick="menuClose()">Program</a>
+          <a href="" onclick="menuClose()">Join</a>
+          <a href="" onclick="menuClose()">News</a>
+      </div>
+    </div>
+  <header class="primary-header">
+    <ul id="social-icons">
+      <li class="media-icons"><img src="./images/facebook.png" alt=""></li>
+      <li class="media-icons"><img src="./images/twitter.png" alt=""></li>
+      <li class="media-icons"><img src="./images/instagram.png" alt=""></li>
+      <li class="media-icons"><img src="./images/linkedin.png" alt=""></li>
+    </ul>
+  </header>
+    <header class="toolbar">
+      <img src="./images/summer.png" alt="" class="summer_img" >
+      <div >
+        <img src='./images/mobile_menu.png' alt="mobile_menu" id="toolbar-img" onclick= menuOpen() >
+      </div>
+      <ul id="link">
+        <li class="links"><a href="index.html">Home</a></li>
+        <li class="links"><a href="">About</a></li>
+        <li class="links"><a href="">Program</a></li>
+        <li class="links"><a href="">Join</a></li>
+        <li class="links"><a href="">News</a></li>
+      </ul>
```

Comment:
- [ ] Kindly fix the navbar links by filling out the `href` attribute in all navbars to make them functional 👍 
- [ ] Kindly fix the design of the navbar in the about page, it doesn't look like the one in the home page and it has empty spaces on top and on the sides.
![Screenshot_1](https://user-images.githubusercontent.com/50721479/190719664-646f1509-5a73-443c-90cb-c67f9f56dcef.png)